_sass/_variables.scss

```diff
@@ -1,6 +1,6 @@
 // Colors
 $color-tc39: #f90;
-$color-primary: #7cb;
+$color-primary: #017e78;
```

Comment:
this should be deleted if we decide that we want to use the same color everywhere. this is open to discussion